src/sto/CarDoor.java

```diff
@@ -0,0 +1,67 @@
+package sto;
+
+public class CarDoor{
+    private boolean closeDoor;
+    private boolean closeWindow;
+
+
+    public CarDoor() {
+        closeDoor=true;
+        closeWindow=true;
+
+    }
+
+    public CarDoor(boolean openDoor, boolean openWindow) {
+        this.closeDoor = openDoor;
+        this.closeWindow = openWindow;
+    }
+
+    public void openDoor(){
+        if (!closeDoor){
```

Comment:
If closeDoor is false it will be false. If it true it will change to false.
Simplification: closeDoor=false;